centaur/src/main/resources/standardTestCases/invalid_mount_reference_disks_specification.test

```diff
@@ -0,0 +1,12 @@
+name: invalid_use_reference_disks_specification
```

Comment:
Suggestion: rename file to `invalid_use_reference_disks_specification.test`

Reply:
oops yes, that should be its name